cmd/qvain-backend/dataset_api.go

```diff
@@ -142,7 +140,7 @@ func (api *DatasetApi) Dataset(w http.ResponseWriter, r *http.Request, user *mod
 			return
 
 		default:
-			jsonError(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
+			loggedJSONError(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed, &api.logger).Msg("Error in Dataset method dispach")
```

Comment:
"Error in Dataset method dispach" This does not bring value on the logs, it is too large. It would make more sense to have some kind of NAME_001 style of line in the log itself, which would not consume space and would be easy to find.

Reply:
The Error message prints the line number of the error. so I think it should be easy to find. Main intention was to log the method information if possible in the error message.

Reply:
I would say that it is kind of an obvious that it is an error, so why to write "Error in" in there. Also would not for example "Unhandled request method (METHOD_HERE)" be more describing. The same applies to all default switch-cases.